### ISSUE TITLE
Adding comments in a review should be accessible 

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode-pull-request-github#1930]
- [x] There should be some screen reader feedback that a line can have a comment added, accessible with mouse and with keyboard.
- [x] There should be a way to add a comment with the keyboard.

**Comment by daniel-montalvo** [comment]
Any progress on this? At least having an action that the user can then bind to their keystroke of choice would be very much appreciated. Of course the ideal situation is that these actions would be already pre-configured with dedicated keystrokes.

**Comment by alex19EP** [comment]
hello. do you plan to work on this? since I cannot leave multi-line comments through the github interface, I would like to do this at least through vsCode.

**Comment by RMacfarlane** [comment]
There is a `workbench.action.addComment` command that can be used for starting a comment, but it doesn't have a keybinding by default, so is extremely undiscoverable. There is also `workbench.action.submitComment` which is bound to ctrl/cmd + enter by default

The screen reader feedback about which lines are commentable is missing

**Comment by alexr00** [comment]
We're still lacking screen reader feedback that a comment can be added on the current line, but other accessibility updates were added a few months ago: https://code.visualstudio.com/updates/v1_66#_comment-accessibility-improvements

**Comment by alexr00** [comment]
When an editor is opened we now give an aria status that the editor has commenting ranges.
